jetty-cdi/cdi-servlet/src/test/java/org/eclipse/jetty/cdi/servlet/WeldInitializationTest.java

```diff
@@ -118,4 +119,11 @@ public void testRequestParamServletAbc() throws Exception
             assertThat("Response", resp, containsString(" param[abc] = [123]"));
         }
     }
+
+    @Test
+    public void testSessionScoped() throws Exception
+    {
+        HttpURLConnection http = (HttpURLConnection) serverHttpURI.resolve("session").toURL().openConnection();
+        assertThat("response code", http.getResponseCode(), is(200));
```

Comment:
It seems odd to just check the status code here.  Is it really just pass/fail, no half-measure?
Does the response at least contain the stacktrace when it does fail? (would help with troubleshooting)

Reply:
A unit test that tests the session scope "persistence" would require an HTTP client that keeps track of session cookies and such. Hence I've kept it simple here, just to capture the NPE.